src/main/resources/findologic_20.xsd

```diff
@@ -191,42 +198,26 @@
             <xs:pattern value="http[s]?://.*"/>
         </xs:restriction>
     </xs:simpleType>
-    <xs:simpleType name="money">
-        <xs:restriction base="xs:decimal">
-            <xs:fractionDigits value="2"/>
-        </xs:restriction>
-    </xs:simpleType>
 
-    <!-- Usergroup support -->
-    <xs:simpleType name="usergroupHash">
```

Comment:
This fieldType allowed usergroups consisting of whitespace only. Doesn't make any sense to me. As there is already a fieldType `nonEmptyString` which can prevent this, i am using this instead. Please let me know if you think different.

Reply:
`nonEmptyString` contains `<xs:whiteSpace value="collapse"/>` which is not the case here, did you check how this will affect the behavior?

Reply:
Having `<xs:whiteSpace value="collapse"/>` actually makes implementing the export easier. Some people have had formatted exports in the past with values like this:

```
    <usergroup>.
........foobert.
....</usergroup>
```

I substituted the white space that would end up as part of the element's value with dots. Customers than complained that they had excessive white space in some places. `<xs:whiteSpace value="collapse"/>` removes all leading and trailing whitespace of element text, as well as all duplicate space within.

If not too excessive, we should consider adding `<xs:whiteSpace value="collapse"/>` to everything to remove this source of pain. As this is something that relaxes the strictness of the format, we could also do that later.

One important thing to note, though: The type specification about collapsing whitespace only applies to validation. When parsing XML, a [special annotation and adapter class](https://stackoverflow.com/a/359363/496143) might be necessary to ensure that surrounding whitespace is trimmed.

Reply:
> we should consider adding <xs:whiteSpace value="collapse"/> to everything to remove this source of pain

I've added this to the collection of open topics, so we do not forget about it. I will handle this in a follow-up
[FINDO-9598](https://findologic.myjetbrains.com/youtrack/issue/FINDO-9598) Open topics XML schema